module/metrics/rest_api.go

```diff
@@ -0,0 +1,137 @@
+package metrics
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/slok/go-http-metrics/metrics"
```

Comment:
maybe give this an alias since this package is also called `metrics`. This should still work as-is, but renaming it would be less confusing.
```suggestion
	httpmetrics "github.com/slok/go-http-metrics/metrics"
```